static/src/javascripts/bootstraps/enhanced/preferences.spec.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import { init } from './preferences.js';
+
+jest.mock('react/addons', () => ({
+    React: {
+        createClass: jest.fn(),
+    },
+}));
+
+const initialiseSummaryTagsSettings = jest.fn();
+
+describe('preferences', () => {
+    it('should exist', () => {
+        expect(init).toBeDefined();
+    });
+
+    it('should not initialise if the placeholder is missing', () => {
+        init();
+        expect(initialiseSummaryTagsSettings).not.toHaveBeenCalled();
```

Comment:
I believe this test will always pass, because the `initialiseSummaryTagsSettings` mock is not linked to the module under test. It might be easier to ensure the `React.createClass` mock has not been called in this test